src/CodeFormatter/CodeFormatter.csproj

```diff
@@ -71,6 +71,10 @@
       <HintPath>..\packages\System.Reflection.Metadata.1.0.21\lib\portable-net45+win8\System.Reflection.Metadata.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="System.Threading.Tasks.Dataflow, Version=4.5.24.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\References\System.Threading.Tasks.Dataflow.dll</HintPath>
```

Comment:
That shouldn't matter, you can just add System.Threading.Tasks.Dataflow to packages.config and reference it like S.R.Metadata above. This way it's no longer an indirect dependency as well.